tests/attacks/poison/test_backdoor_attack.py

```diff
@@ -19,40 +19,31 @@
 
 import os
 import logging
-import unittest
 
 import numpy as np
+import pytest
 
 from art.attacks.poisoning.backdoor_attack import PoisoningAttackBackdoor
 from art.attacks.poisoning.perturbations import add_pattern_bd, add_single_bd, insert_image
 from art.utils import to_categorical
-
-from tests.utils import TestBase, master_seed, get_image_classifier_kr
+from tests.utils import ARTTestException
 
 logger = logging.getLogger(__name__)
 
 PP_POISON = 0.33
 NB_EPOCHS = 3
 
+backdoor_path = os.path.join(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))),
+                             "utils", "data", "backdoors", "alert.png")
 
-class TestBackdoorAttack(TestBase):
-    """
-    A unittest class for testing Backdoor Poisoning attack.
-    """
+max_val = 1
+
+
```

Comment:
Should we add checks to this file to make sure calls to `fit` make expected changes?

Reply:
Same as above. Additionally, I think testing that the poison is effective is best showed in the accompanying notebook since the process of poisoning to completion may take more data/epochs than is used in our test runs.

Reply:
Same as above, let's continue the dicsussion there.